os/kernel/task/task_dumptasks.c

```diff
@@ -139,14 +139,15 @@ static void task_taskdump(FAR struct tcb_s *tcb, FAR void *arg)
 
 void task_show_alivetask_list(void)
 {
-	lldbg("*******************************************\n");
-	lldbg("List of all tasks in the system:\n");
-	lldbg("*******************************************\n");
+	lldbg_noarg("---------------------------------------------------------------------------------------------------\n");
```

Comment:
With this change, TRAP is working? Could you check this with @thapav ?

Reply:
Sure I will check it once merged.